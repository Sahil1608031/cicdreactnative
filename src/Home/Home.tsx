import React, { JSX, useState } from 'react';
import { View, Text, StyleSheet } from 'react-native';
//home compoent
const Home = (): JSX.Element => {
  const [data, setData] = useState<string>('');

  return (
    <View style={styles.container}>
      <Text>Home</Text>
    </View>
  );
};
const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
});
export default Home;
