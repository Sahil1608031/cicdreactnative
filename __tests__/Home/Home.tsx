import React from "react";
import { render } from "@testing-library/react-native";
import Home from "../../src/Home/Home";

describe("Home Component", () => {
    it("renders correctly", () => {
        const { getByText } = render(<Home />);
        expect(getByText("Home")).toBeTruthy();
    });

    it("matches snapshot", () => {
        const tree = render(<Home />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
