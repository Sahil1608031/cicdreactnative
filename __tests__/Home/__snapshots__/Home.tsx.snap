// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component matches snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Home
  </Text>
</View>
`;
